### PR TITLE
[FIX] typo, tray_id doesn't exist only printer_tray_id exist on the o…

### PR DESCRIPTION
**Pull request by vrenaville**
…bject

### CODE REVIEW

**Comment by gurneyalex** [comment]
:+1:

**Comment by damdam-s** [comment]
:+1:

File: printer_tray/printer.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def print_options(self, report, format):
                                           ('action', '!=', 'user_default')],
                                          limit=1)
         if action and action.printer_tray_id:
```
Comment (gurneyalex) [inline printer_tray/printer.py:95]: in v8, you can just say:

```
if action.printer_tray_id:
```

**Comment by pedrobaeza** [comment]
:+1: with the change suggested by Alexandre

**Comment by yvaucher** [comment]
fixed the comment in the merge

**Pull request merged**